pages/manual/manual-gettingstarted-build.md

```diff
@@ -29,6 +29,13 @@ The `--recurse-submodules` is necessary to pull dependencies.
  Use front slashes and no space between modules. Configure again. External modules are developed in the [inviwo modules repository](https://github.com/inviwo/modules).
 4. Hit Generate and open the project in your IDE.
 
+{% include note.html content="
+When using a multi-configuration generator (like Visual Studio and most IDEs) you may want to adjust your build mode manually, since it probably defaults to `Debug` at first, which has a large impact on the performance.
+If you use a single-configuration generator, you can control the build mode using `CMAKE_BUILD_TYPE` in CMake, which defaults to `Release`
```

Comment:
I'm not sure if it defaults to release. We could just link to:
https://cmake.org/cmake/help/latest/variable/CMAKE_BUILD_TYPE.html

Reply:
I have looked into it a bit. the default is empty and apparently empty defaults to Release (according to some stack overflow pages.. could not find any official information on this")

Reply:
more investigation actually gave some more insight. Leaving this empty defaults to build mode without optimization and without debug symbols. We should just set a default to this property for all single-generator users... I'll adjust this accordingly..